source/_examples/configuration_yaml_by_silvrrgit.markdown

```diff
@@ -1,6 +0,0 @@
----
-title: "Configuration.yaml by SilvrrGIT"
-description: ""
-ha_category: Example configuration.yaml
```

Comment:
Why is this change in your PR?

Reply:
I had my local branch based on the default (`current`) but opened up the PR against `next`. I just rebased and force pushed up a change to fix this